lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +20,22 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str) or len(text) < 1:
+        return None
+
+    for symbol in text:
+        if symbol.isalpha() or symbol == " ":
+            continue
+        text = text.replace(f"{symbol}", " ")
+
+    new_text = text.split(" ")
```

Comment:
Is this really necessary to create new_text?

Reply:
fixed